query-languages/m/m-spec-error-handling.md

```diff
@@ -55,40 +55,48 @@ Raising an error will cause the current expression evaluation to stop, and the e
 
 ## Handling errors
 
-An _error-handling-expression_ is used to handle an error:
+An _error-handling-expression_ (informally known as a "`try` expression") is used to handle an error:
```

Comment:
The formatting of the back-tick inside the quote doesn't look quite right. Consider removing the backticks.

Reply:
Done